src/pimlify-vue/src/components/RestaurantComponent.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+<v-card v-if="restaurants !== undefined">
+    <v-card-title class="justify-center">
+        <b>Restaurants</b>
+    </v-card-title>
+    <v-data-table
+    :items="restaurants"
+    :headers="headers"
+    class="elevation-1"
+    hide-actions
+  >
+    <template slot="items" slot-scope="props">
+      <td>{{ props.item.id }}</td>
+      <td>{{ props.item.name }}</td>
+      <td>{{ props.item.location }}</td>
+
+    </template>
+  </v-data-table>
+</v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import Component from 'vue-class-component';
+
+@Component
+export default class RestaurantComponent extends Vue {
+  data() {
```

Comment:
Could simply be public headers without data

Reply:
Isn't it best practice in vue components to use data?